### PR TITLE
Walk Reluctance Checkbox

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
Changes default walk reluctance input from slider to checkbox.

### CODE REVIEW

**Review by daniel-heppner-ibigroup — APPROVED** [review]
per discussion, let's remove the override thing.

File: packages/trip-form/i18n/en-US.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ otpUi:
       rail-label: Train
       subway-label: Subway
       tram-label: Tram
+      walkReluctance-label: Avoid Walking?
```
[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply packages/trip-form/i18n/en-US.yml:28]: do we hate

[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply packages/trip-form/i18n/en-US.yml:28]: yes

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Great fix, really clean! A little confused about the `subsettingOverrides` and the question mark.

File: packages/trip-form/i18n/en-US.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply packages/trip-form/i18n/en-US.yml:28]: wow...

File: packages/trip-form/src/MetroModeSelector/index.tsx [hunk 2]
```diff
@@ -399,6 +402,11 @@ interface Props {
    * @param key Mode button to be toggled
    */
   onToggleModeButton: (key: string, newState: boolean) => void;
+  /**
+   * Array of configurable overrides for mode subsettings. For example, walk reluctance defaults
+   * to checkmark but slider is also supported.
+   */
+  subsettingOverrides?: Array<ModeSetting>;
```
[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply packages/trip-form/src/MetroModeSelector/index.tsx]: No! Just missed taking out all the prop drilling in [ab1cae8](https://github.com/opentripplanner/otp-ui/pull/631/commits/ab1cae8087f800be8959a2fe205650c2178f9902)! Removed in [f26ce83](https://github.com/opentripplanner/otp-ui/pull/631/commits/f26ce83d9d46a88ea9e3f0bfe9b85c640b8b39c7)

File: packages/trip-form/src/MetroModeSelector/index.tsx [hunk 3]
```diff
@@ -399,6 +399,10 @@ interface Props {
    * @param key Mode button to be toggled
    */
   onToggleModeButton: (key: string, newState: boolean) => void;
+  /**
+   * Array of configurable overrides for mode subsettings. For example, walk reluctance defaults
+   * to checkmark but slider is also supported.
+   */
```
Comment (miles-grant-ibigroup) [inline packages/trip-form/src/MetroModeSelector/index.tsx:405]: I think you forgot to take out this comment!

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Just one more small thing to remove and then if you could please merge master into this branch so we have a cleaner merge we should be good to go!

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks wonderful thanks!

**Pull request merged**